diff -urN pinfo-0.6.8/src/video.c pinfo-0.6.8.patch/src/video.c
--- pinfo-0.6.8/src/video.c	2002-07-21 20:42:00.000000000 +0200
+++ pinfo-0.6.8.patch/src/video.c	2004-09-25 05:20:10.042759936 +0200
@@ -66,7 +66,7 @@
   attrset (normal);
   for (i = pos; (i < lines) && (i < pos + maxy - 2); i++)
     {
-      int tmp = strlen (message[i]) - 1;
+      int tmp = strlen (message[i]);
       message[i][tmp] = 0;
       if(tmp>column)
         mvaddstr (i + 1 - pos, 0, message[i]+column);
