--- pinfo-0.6.3/src/Makefile.am.orig	Sat Oct 20 14:56:33 2001
+++ pinfo-0.6.3/src/Makefile.am	Sat Oct 20 14:58:10 2001
@@ -67,11 +67,6 @@
 pinfo_LDADD = $(pinfo_READLINE_OBJ) $(pinfo_SNPRINTF_OBJ)\
               $(pinfo_SIGBLOCK_OBJ) @READLINE_LIBS@ $(INTLLIBS)
 
-pinfo_DEPENDENCIES = $(pinfo_READLINE_OBJ)\
-                     $(pinfo_SNPRINTF_OBJ)\
-                     $(pinfo_SIGBLOCK_OBJ)\
-		     $(INTLDEPS)
-
 
 if HAS_READLINE
 else
