--- pinfo-0.6.13/src/Makefile.am.orig	2021-10-01 21:09:56.178358747 +0200
+++ pinfo-0.6.13/src/Makefile.am	2021-10-01 21:10:29.424845302 +0200
@@ -68,11 +68,6 @@ pinfo_CFLAGS = $(CURSES_CFLAGS) $(WCHAR_
 	$(READLINE_INCLUDES) $(WFLAGS) \
 	-DLOCALEDIR=\"$(LOCALEDIR)\" -DCONFIGDIR=\"$(sysconfdir)/pinforc\"
 
-pinfo_DEPENDENCIES = $(pinfo_READLINE_OBJ)\
-                     $(pinfo_SNPRINTF_OBJ)\
-                     $(pinfo_SIGBLOCK_OBJ)\
-                     $(INTLDEPS)
-
 
 if HAS_READLINE
 else
